### DISCUSSION TITLE
Error while applied the command npx shadcn@latest add scroll-area

### DISCUSSION THREAD

**Discussion by mailmonir** [shadcn-ui/ui#4865] [General]
Got the following error

Something went wrong. Please check the error below for more details.
If the problem persists, please open an issue on GitHub.

Command failed with exit code 1: npm install tailwindcss-animate class-variance-authority lucide-react @radix-ui/react-icons clsx tailwind-merge
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @uploadthing/react@6.7.2
npm error Found: uploadthing@6.13.3
npm error node_modules/uploadthing
npm error   uploadthing@"^6.13.3" from the root project
npm error
npm error Could not resolve dependency:
npm error peer uploadthing@"6.13.2" from @uploadthing/react@6.7.2
npm error node_modules/@uploadthing/react
npm error   @uploadthing/react@"^6.7.2" from the root project
npm error
npm error Conflicting peer dependency: uploadthing@6.13.2
npm error node_modules/uploadthing
npm error   peer uploadthing@"6.13.2" from @uploadthing/react@6.7.2
npm error   node_modules/@uploadthing/react
npm error     @uploadthing/react@"^6.7.2" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\MIslam\AppData\Local\npm-cache\_logs\2024-09-16T04_52_36_058Z-eresolve-report.txt

npm error A complete log of this run can be found in: C:\Users\MIslam\AppData\Local\npm-cache\_logs\2024-09-16T04_52_36_058Z-debug-0.log

**Comment by Abolfazl-ghodrati-k** [comment]
no answers?
im facing same issue on adding a component to project
currently using next15 and next auth and they have conflicts in using react

is there any way to force installation?

**Comment by SilasKierstein** [comment]
try running "npx --legacy-peer-deps shadcn@latest init -d" to force install

**Reply by ademsuslu** [reply]
Ok thanks, the problem is solved, but what is the reason for the error?
